Documentation/ceph-disaster-recovery.md

```diff
@@ -237,6 +238,109 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
 
+## Restoring Mon Quorum using healthy OSDs
+
+If all mons fail, it might be able to recover monstore from OSDs. [Ceph's document](https://docs.ceph.com/en/latest/rados/troubleshooting/troubleshooting-mon/#recovery-using-osds) describes how to do it.
```

Comment:
```suggestion
If all mons fail, it might be possible to recover the monstore from the OSDs. [Ceph's documentation](https://docs.ceph.com/en/latest/rados/troubleshooting/troubleshooting-mon/#recovery-using-osds) describes how to do it.
```